### ISSUE TITLE
TypeError: x.get is not a function

### ISSUE THREAD

**Issue by ptitzler** [ibm-watson-data-lab/pixiedust_node#26]
We saw the following error intermittently:

<img width="1103" alt="screen shot 2018-02-09 at 11 13 32 am" src="https://user-images.githubusercontent.com/13068832/36045906-09c406f8-0d8c-11e8-98ee-223b31d4d0ef.png">

Sequence:

```
%%node
// connect to Cloudant using cloudant-quickstart
var cqs = require('cloudant-quickstart');
var cities = cqs('https://...-bluemix.cloudant.com/cities');
```

[works]
```
%%node
cities.get('2636749').then(print).catch(print);
```

[fails sometimes; see screen cap]
```
%%node
cities.get(['4562407','2636749','3530597']).then(print).catch(print);
```

Once the error was encountered, `cities` is rendered unusable (no method calls work anymore)

**Comment by glynnbird** [comment]
I think I know what's going on here! Because `cities` is a global variable it is being sent to and from the Node.js & Python environments.

If you do 

```
const cqs = require('cloudant-quickstart');
const cities = cqs('https://...-bluemix.cloudant.com/cities');
```

that should fix it i.e. swap `var` for `const`.

**Comment by ptitzler** [comment]
Yup, that did it. If this is a general "limitation"/recommendation we should probably document it.

**Comment by glynnbird** [comment]
Fixed in 0.2.4. Objects that contain functions are not migrated from Node->Python.
